### PR TITLE
* add twitter action

### PR DESCRIPTION
**Pull request by hjanuschka**
lets you post on twitter.

``` ruby
twitter(
      access_token: "xxx",
      access_token_secret: "xxx",
      consumer_key: "xxx",
      consumer_secret: "xxx",
      message: "#iosprocessingtime 6 min."
  )
```

### CODE REVIEW

File: docs/Actions.md [hunk 1]
```diff
@@ -1746,6 +1746,23 @@ It can process unit test results if formatted as junit report as shown in [xctes
 
 ## Misc
 
+### twitter
+
+Post a tweet on twitter.
+requires you to setup an app on twitter.com and obtain consumer and access_token
+
+
```
Comment (ohwutup) [inline docs/Actions.md]: I know that these don't render in the HTML, but would you mind just cleaning up at least one of these spaces? :smile:

**Comment by ohwutup** [comment]
This looks great! Thanks @hjanuschka! Just one comment and we will merge away! :octopus:

**Comment by hjanuschka** [comment]
* changes made as suggested
 * rebased current master and squashed PR to one commit


happy merging @ohwutup

**Comment by ohwutup** [comment]
:boom: :+1: :octopus:

**Pull request merged**